docs/how-to/code/install-slice/rockcraft.yaml

```diff
@@ -1,6 +1,6 @@
 name: custom-openssl-rock
 base: bare
-build_base: "ubuntu@22.04"
```

Comment:
a side-effect of this change is that I found out that we apparently accept `build_base` as well as `build-base`
weird